packages/clarity-js/src/layout/dom.ts

```diff
@@ -307,6 +307,12 @@ function updateSelector(value: NodeValue): void {
     }
 }
 
+export function hashText(hash: string): string {
+    let id = lookup(hash);
+    let node = getNode(id);
+    return node != null ? node.textContent : '';
```

Comment:
Only look at a substring with max length of 20 or something reasonable so we accidentally don't send back the whole text.